### PR TITLE
史胜帆week03作业

### PR DESCRIPTION
**Pull request by Jackkcaj7**
### 问题：loss最低达到1.6左右 不知为何无法收敛

### CODE REVIEW

**Comment by Jackkcaj7** [comment]
## week04 sklearn.datasets.fetch_olivetti_faces数据集合训练
### batchnorm dropout adam等新方法的使用
### kaggle免费算力使用

**Comment by Linlaoshi-nlp** [comment]
> ## KMNIST数据集下载与训练并预测
> ### 问题：loss最低达到1.6左右 不知为何无法收敛

一般隐藏层不使用sigmoid，会造成梯度消失的问题，结合作业答案尝试修改下模型结构，多尝试
